constants/API_URL.js

```diff
@@ -1,9 +1,9 @@
-export const API_URL = 'https://impulsoapi.herokuapp.com/'
+export const API_URL = 'https://api-usuarios-feat-prime-biejkp.herokuapp.com/'

```

Comment:
@dlopes14 as requisições estão usando as URLs das APIs de staging
```suggestion
export const API_URL = 'https://impulsoapi.herokuapp.com/'

```
Temos que lembrar de mergear as mudanças na API de usuário na segunda-feira